test/helpers/selenium-helper.js

```diff
@@ -1,4 +1,4 @@
-jasmine.DEFAULT_TIMEOUT_INTERVAL = 30000; // eslint-disable-line no-undef
+jest.setTimeout(30000); // eslint-disable-line no-undef
```

Comment:
Confirmed that these are equivalent via https://jestjs.io/docs/en/jest-object.html#jestsettimeouttimeout and https://github.com/facebook/jest/issues/652